modules/travel_pay/app/controllers/travel_pay/v0/complex_claims_controller.rb

```diff
@@ -10,7 +10,30 @@ class ComplexClaimsController < ApplicationController
       rescue_from Common::Exceptions::BadRequest, with: :render_bad_request
       rescue_from Common::Exceptions::ServiceUnavailable, with: :render_service_unavailable
 
-      before_action :check_feature_flag, only: [:create]
+      before_action :check_feature_flag
+
+      def submit
+        verify_feature_flag!(
+          :travel_pay_enable_complex_claims,
+          current_user,
+          error_message: 'Travel Pay submit complex claim unavailable per feature toggle'
+        )
+
+        claim_id = params[:claim_id]
+        validate_claim_id_exists!(claim_id)
+
+        # TODO: add validation to verify there is a document associated to a given expense
+        # TODO: possibly add validation to verify the claim id is valid
+        Rails.logger.info(message: 'Submit complex claim')
+        submitted_claim = claims_service.submit_claim(claim_id)
+
+        render json: submitted_claim, status: :created
+      rescue ArgumentError => e
+        raise Common::Exceptions::BadRequest.new(detail: e.message)
+      rescue Faraday::ClientError, Faraday::ServerError => e
+        Rails.logger.error("Faraday error submitting complex claim: #{e.message}")
+        raise Common::Exceptions::InternalServerError.new(exception: e)
```

Comment:
With this rescue, we're mapping client errors to internal server errors which may be confusing. 2 thoughts:

1. We should separate the ClientError handling from the ServerError handling
2. We need to talk to the API team to see what status code will come back from a submission fails the "at least 1 doc attached" validation
3. It is fine to change this here only, no need to go fix this elsewhere